### PR TITLE
Fix typos in common.sh

### PR DESCRIPTION
**Pull request by Maxi-Mega**
Signed-off-by: Maxi_Mega <52792549+Maxi-Mega@users.noreply.github.com>

# Comprehensive Summary of your change

- Fix two typos in make/common.sh

Please indicate you've done the following:
- [X] Well Written Title and Summary of the PR
- [ ] Label the PR as needed. "release-note/ignore-for-release, release-note/new-feature, release-note/update, release-note/enhancement, release-note/community, release-note/breaking-change, release-note/docs, release-note/infra, release-note/deprecation"
- [x] Accepted the DCO. Commits without the DCO will delay acceptance.
- [ ] Made sure tests are passing and test coverage is added if needed.
- [ ] Considered the docs impact and opened a new docs issue or PR with docs changes if needed in [website repository](https://github.com/goharbor/website).

### CODE REVIEW

File: make/common.sh [hunk 1]
```diff
@@ -133,4 +133,4 @@ function check_dockercompose {
 		error "Failed to parse docker-compose version."
 		exit 1
 	fi
-}
\ No newline at end of file
+}
```
Comment (OrlinVasilev) [inline make/common.sh:136]:
```suggestion
}

```

**Review by OrlinVasilev — CHANGES_REQUESTED** [review]
Hi @Maxi-Mega  thanks for considering contribution to our project :) please add new line at the end!

Thank you!

**Review by OrlinVasilev — APPROVED** [review]
lgtm

**Review by wy65701436 — APPROVED** [review]
lgtm

**Pull request merged**